apps/user-profile/src/components/common/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { useState, useEffect } from "react";
+import ProgressBarStyle from './StyledOnboardingProgressBar';
+
+
+const ProgressBar = ({
+    bgColor,
```

Comment:
The only property that the progressBar should take at the very moment is `percentage`.

Reply:
Okay

Reply:
It is but I have also used the property `className`